src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -199,7 +199,7 @@ private void OnCurrentWorkspaceChanged(IWorkspaceModel ws)
 
         private PackageDependencyInfo GetNodePackageFromAssemblyName(AssemblyName assemblyName)
         {
-            if (NodePackageDictionary.ContainsKey(assemblyName.FullName))
+            if (NodePackageDictionary!= null && NodePackageDictionary.ContainsKey(assemblyName.FullName))
```

Comment:
does it also get initialized to an empty dictionary somewhere? Like in the constructor?

Reply:
Yeah, it get's initialized when the first package is loaded